frontend/src/libs/components/survey-category/survey-category.tsx

```diff
@@ -0,0 +1,45 @@
+import { useCallback } from '#libs/hooks/hooks.js';
```

Comment:
should be inside pages/survey/components

Reply:
Done

Reply:
Done